tests/dft-grad2/input.dat

```diff
@@ -1,9 +1,10 @@
 #! DF-BP86-D2 cc-pVDZ frozen core gradient of S22 HCN
-
+#! updated ref gradient due to new BraggSlater radii
+   
 ref = psi4.Matrix.from_list([                                       #TEST
-    [ 0.01503184898954, -0.01251384566028, 0.00000000000000],       #TEST
-    [ 0.01060215774396, -0.00433207404216, 0.00000000000000],       #TEST
-    [-0.02563442179393,  0.01685678274170, 0.00000000000000]        #TEST
+    [ 0.015033129924,    -0.012524858341,     0.000000000000],       #TEST
+    [ 0.010601799255,    -0.004332928749,     0.000000000000],       #TEST
+    [-0.025634252349,     0.016856185932,     0.000000000000]        #TEST
```

Comment:
Did you generate the new ref vals with tight convcrit? There's a few guidelines in the last paragraph of [here](http://psicode.org/psi4manual/master/add_tests.html). Or ask for any advice. Otherwise problems can arise on different computers just due to variation in number of iterations, etc.

Reply:
no, i did not. Will have to change that. I used the settings from the tests themselves. I assume this is mainly an issue of the gradient tests?

Reply:
It hits energy tests, too, unfortunately. set e_/d_/r_convergence to 10, collect the ref numbers, remove the convergences, and tests should be good to go.

Reply:
updated the tests with tighter convergence.